lib/components/appbar.dart

```diff
@@ -8,7 +9,400 @@ class GlobalAppBar extends StatefulWidget implements PreferredSizeWidget  {
+        elevation: 7.0,
+        child: Center(
+          child: Text(
+            'Settings',
+            style: TextStyle(
+              color: Colors.black,
+              fontFamily: 'B612'
+              ),
+            ),
+         ),
+        ),
+      ),  
+      onTap: () {  
+      showSettingsDialog(context);  
+     }, 
+    ), 
+  );
+  Widget profile=Container(
+        margin: const EdgeInsets.all(2.0),
+    decoration:BoxDecoration(
+      borderRadius: BorderRadius.circular(10.00),
+      border: Border.all( 
+        color: Color(0xFF6ad1ff)
+        )
+      ),
+    child:GestureDetector(
+      child:Container(
+        height: 50.0,
+        child: Material(
+        borderRadius: BorderRadius.circular(10.0),
+        shadowColor: Color(0xFF6ad1ff),
+        //color: Color(0xFF039be5),
+        elevation: 7.0,
+        child: Center(
+          child: Text(
+            'Profile',
+            style: TextStyle(
+              color: Colors.black,
+              fontFamily: 'B612'
+              ),
+            ),
+         ),
+        ),
+      ),  
+      onTap: () {  
+      Navigator.of(context).pop();  
+     }, 
+    ), 
+  );
+    Widget about=Container(
+        margin: const EdgeInsets.all(2.0),
+    decoration:BoxDecoration(
+      borderRadius: BorderRadius.circular(10.00),
+      border: Border.all( 
+        color: Color(0xFF6ad1ff)
+        )
+      ),
+    child:GestureDetector(
+      child:Container(
+        height: 50.0,
+        child: Material(
+        borderRadius: BorderRadius.circular(10.0),
+        shadowColor: Color(0xFF6ad1ff),
+        //color: Color(0xFF039be5),
+        elevation: 7.0,
+        child: Center(
+          child: Text(
+            'About',
+            style: TextStyle(
+              color: Colors.black,
+              fontFamily: 'B612'
+              ),
+            ),
+         ),
+        ),
+      ),  
+      onTap: () {  
+      showAboutDialog(context);  
+     }, 
+    ), 
+  );
+  Widget howtoplay=Container(
+    margin: const EdgeInsets.all(2.0),
+    decoration:BoxDecoration(
+      borderRadius: BorderRadius.circular(10.00),
+      border: Border.all( 
+        color: Color(0xFF6ad1ff)
+        )
+      ), //             <--- BoxDecoration here
+    child:GestureDetector(
+      child:Container(
+        height: 50.0,
+        child: Material(
+        borderRadius: BorderRadius.circular(10.0),
+        shadowColor: Color(0xFF6ad1ff),
+        //color: Color(0xFF039be5),
+        elevation: 7.0,
+        child: Center(
+          child: Text(
+            'How To Play',
+            style: TextStyle(
+              color: Colors.black,
+              fontFamily: 'B612'
+              ),
+            ),
+         ),
+        ),
+      ),  
+      onTap: () {  
+      Navigator.of(context).pop();  
+     }, 
+    ), 
+  );
+  AlertDialog alert = AlertDialog(  
+    actions: [  
+      title, profile, settings, howtoplay, about,
+    ],  
+  );  
```

Comment:
<img width="297" alt="Dialog Box" src="https://user-images.githubusercontent.com/26324376/88454782-72ce7680-ce8f-11ea-88aa-e61b563f3b38.png">

Some design improvements:
- Give rounded border BoxDecoration
- Add little more space between the items, add space from left and right.
- "Options" is not centrally aligned
- Add more space for the content (make dialog bigger)
-